react/FilePathLink/styles.styl

```diff
@@ -0,0 +1,6 @@
+.c-file-path-link
+  text-decoration: none
+
+  :first-child:hover 
+    color var(--secondaryTextColor)
+    text-decoration underline
```

Comment:
Can't we use the MuiTheme to customize this component?

Reply:
Are we talking about the text-decoration part? the inverted theme works as expected for other properties 
![image](https://user-images.githubusercontent.com/12577784/130617554-1d352c69-a471-41a4-a7e2-0ffff758468a.png)

Reply:
Je parle pas de ça. 

Link est un composant MUI. Tous les composants MUI se basent sur la définition du thème : https://github.com/cozy/cozy-ui/blob/master/react/MuiCozyTheme/theme.jsx#L190 . On devrait ne pas ajouter de stylus pour ce composant, on devrait ajouter l'entrée MuiLink (ou cf doc api j'ai pas en tête le nom) dans le thème.

Reply:
ok override le underline des link material ui dès le theme je comprends. Cela ne risque-t-il pas de changer le style dans les apps qui utilisent déjà Link de cozy-ui (qui n'override pas mui pour l'instant) ?